src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -73,14 +73,11 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
   };
 
   const getSubstratesAccordingToType = (type?: string) => {
-    if (type) {
-      if (type === 'Lab') {
-        return LAB_SUBSTRATES;
-      } else {
-        return NURSERY_SUBSTRATES;
-      }
-    }
-    return [];
+    if (type === 'Lab') {
+      return LAB_SUBSTRATES;
+    } else if (type === 'Nursery') {
+      return NURSERY_SUBSTRATES;
+    } else return [];
```

Comment:
Not sure if lint will flag this but feel free to merge when ready. Thanks for following up 🎉